### ISSUE TITLE
Incompatibility with latest Azure CLI

### ISSUE THREAD

**Issue by sdebruyn** [terraform-providers/terraform-provider-azurerm#8033]
### Community Note

* Please vote on this issue by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue to help the community and maintainers prioritize this request
* Please do not leave "+1" or "me too" comments, they generate extra noise for issue followers and do not help prioritize the request
* If you are interested in working on this issue or have submitted a pull request, please leave a comment

### Terraform (and AzureRM Provider) Version
```
Terraform v0.12.29
+ provider.azurerm v2.21.0
```

### Affected Resource(s)
* all resources

### Terraform Configuration Files
Anything with Azure CLI authentication

### Debug Output
```
Error: Error obtaining Authorization Token from the Azure CLI: Error parsing json result from the Azure CLI: Error waiting for the Azure CLI: exit status 1
```

### Expected Behavior
Authentication works

### Actual Behavior
Error above

### Steps to Reproduce
1. Update to Azure CLI 2.10.0
1. `terraform init`

### Important Factoids
Using a remote back-end on Azure

**Comment by tombuildsstuff** [comment]
This'll be fixed via https://github.com/hashicorp/terraform/pull/25769

**Comment by tombuildsstuff** [comment]
Closing since https://github.com/hashicorp/terraform/pull/25769 has been merged (which should fix this) - which'll go out in TF Core 0.13.1
